tests/conftest.py

```diff
@@ -254,14 +254,16 @@ def init_app(self, app, db):
 def app(request, versionfile, docker_env_vars, disable_migrations, mocked_repo_config):
     """Needed for pytest-flask."""
     module = request.module
-    spec = "swagger.yml" if not hasattr(module, "TREESTATUS_APP") else "treestatus.yml"
 
     config = load_config()
+    config["API_SPEC"] = (
+        "swagger.yml" if not hasattr(module, "TREESTATUS_APP") else "treestatus.yml"
```

Comment:
Same thing here, we shouldn't only be checking if the attribute is there or not, we should be checking the value of the attribute. However, this also circumvents the functionality in `load_config`, which should be tested. Instead, `TREESTATUS_APP` should be monkeypatched in ` docker_env_vars`, similar to how other environment variables are set.

Reply:
Good call, I had the config knob as `API_SPEC` in one location and `APP_SPEC` in another, so this already saved me time debugging in dev.